Source/Core/Proj4Projection.js

```diff
@@ -0,0 +1,196 @@
+define([
+        './Cartesian3',
+        './Cartographic',
+        './Math',
+        './Check',
+        './defaultValue',
+        './defined',
+        './defineProperties',
+        './Ellipsoid',
+        './oneTimeWarning',
+        './Rectangle',
+        '../ThirdParty/proj4-2.5.0'
+    ], function(
+        Cartesian3,
+        Cartographic,
+        CesiumMath,
+        Check,
+        defaultValue,
+        defined,
+        defineProperties,
+        Ellipsoid,
+        oneTimeWarning,
+        Rectangle,
+        proj4) {
+    'use strict';
+
+    // proj4 may crash with absolute extents
+    var smallerThanMax = Rectangle.fromDegrees(-180.0 + CesiumMath.EPSILON7, -90.0 + CesiumMath.EPSILON7, 180 - CesiumMath.EPSILON7, 90 - CesiumMath.EPSILON7);
+
+    /**
+     * MapProjection using proj4js. This projection is only to be used with Ellipsoid.WGS84.
+     * Users should exercise caution when using local-area projections, as local area projections
```

Comment:
cautious how?  What can they do to avoid issues?  I think we need to write this doc with the user in mind.  As spoken about in another comment, remove all mention of `proj4js` from the actual documentation or API, it's a leaky abstraction.